templates/index.html

```diff
@@ -216,8 +216,10 @@ <h3><u>Contact Us</u></h3>
         })
         .catch(error => {
           console.log(error);
+          text = 'Something went wrong. Please try again or contact help@rememberthee.com.';
+          error_message.innerHTML = text;
```

Comment:
we also probably need to undo the styling done in lines 192-196 above so that the user can submit the button to try again.